A##-list-health.md

```diff
@@ -0,0 +1,59 @@
+# Add List endpoint to gRPC Health service
+
+* **Author(s):** Marcos Huck
+* **Approver:** TBD
+* **Status:** Draft  
+* **Implemented in:** TBD
+* **Last updated:** 2024-12-17  
+* **Discussion at:** TBD
+
+
+## Abstract
+
+This proposal introduces a new `List` RPC endpoint for the Health service, allowing clients to retrieve the statuses of all monitored services or a filtered list of specific services. This feature simplifies integration with status-reporting dashboards and enhances observability for microservices.
```

Comment:
What we're talking about adding here is a method, not an endpoint.  Please update terminology throughout.